streamz/tests/test_core.py

```diff
@@ -65,7 +71,10 @@ def f(acc, x, y=None):
 
 def test_filter():
     source = Stream()
-    L = source.filter(lambda x: x % 2 == 0).sink_to_list()
+    sout = source.filter(lambda x: x % 2 == 0, stream_name="filtering")
+    L = sout.sink_to_list()
+
+    assert sout.name == "filtering"
```

Comment:
It might be nice to avoid having these checks in all of the normal tests.  I understand the objective to test that all of the different Stream subclasses pass through kwargs.  Maybe this can be tested in a single isolated test rather than in all of the tests?

Reply:
I agree, I think it is more organized, and less painful to go back and read. Changed it. thanks!